src/Indicator.vala

```diff
@@ -54,11 +54,10 @@ public class DateTime.Indicator : Wingpanel.Indicator {
             settings_button.text = _("Date & Time Settings…");
 
             main_grid = new Gtk.Grid ();
-            main_grid.halign = Gtk.Align.CENTER;
-            main_grid.valign = Gtk.Align.START;
-            main_grid.attach (calendar, 0, 0);
-            main_grid.attach (new Wingpanel.Widgets.Separator (), 0, 2);
-            main_grid.attach (settings_button, 0, 3);
+            main_grid.hexpand = false;
```

Comment:
hexpand is false by default, so no need for this line

Reply:
```suggestion
```